Gradata/tests/test_inject_handoff_hook.py

```
@@ -0,0 +1,149 @@
+
+        def fake_emit(event_type, source, data=None, tags=None, **kw):
+            del kw
+            calls.append((event_type, data or {}))
+
+        from gradata import _events as events
+
+        monkeypatch.setattr(events, "emit", fake_emit)
+
+        inject_handoff.main({})
+        assert any(c[0] == "handoff.injected" for c in calls)
+        injected = [c for c in calls if c[0] == "handoff.injected"][0][1]
```

Ruff (0.15.10)

[warning] Prefer `next(c for c in calls if c[0] == "handoff.injected")` over single element slice

Replace with `next(c for c in calls if c[0] == "handoff.injected")`

(RUF015)